src/Command.ts

```diff
@@ -1656,8 +1656,8 @@ export default class Command<T, P = any>
    * @returns The value of the attribute, or `null` if no such attribute
    * exists.
    */
-  getAttribute<T = any>(name: string) {
-    return this._callElementMethod<T>('getAttribute', name);
+  getAttribute(name: string) {
```

Comment:
Can we leave the generic, but have it default to `StringResult<T>` instead of `any`? We don't actually need the generic, but removing out would cause problems for users that had been using it.